include/retdec/cpdetect/compiler_detector/heuristics/pe_heuristics.h

```diff
@@ -13,6 +13,12 @@
 namespace retdec {
 namespace cpdetect {
 
+struct PeHeaderStyle
+{
+	const char * headerStyle;
```

Comment:
Do we need to use bare `const char *`? Wouldn't `std::string` be safer? Just to consider. The code currently works fine even with `const char *`.

Reply:
Unfortunately, having "(const) std::string" instead of "const char *" here makes MS Visual Studio 2017 compiler (v 15.9.8) exit with `fatal error C1001: An internal error has occurred in the compiler`

Reply:
OK, you can keep the code as-is.